frontend/src/scenes/dashboard/dashboardLogic.tsx

```diff
@@ -385,10 +390,14 @@ export const dashboardLogic = kea<dashboardLogicType>({
     }),
     events: ({ actions, cache, props }) => ({
         afterMount: () => {
-            actions.loadDashboardItems({
-                refresh: props.internal,
-                dive_source_id: dashboardsModel.values.diveSourceId ?? undefined,
-            })
+            if (props.id) {
```

Comment:
added a test to prove this change fixes the issue

Reply:
That's awesome, thanks Paul!